DistroLauncher/console_service.h

```diff
@@ -11,11 +11,12 @@ namespace Win32Utils
         HANDLE stdErrHandle = nullptr;
         // The identity of this state is embedded into the values of the handles.
         // File descriptor values will most likely be always new since they are created by calling _dup().
-        friend bool operator==(const ConsoleState& lhs, const ConsoleState& rhs)
-        {
-            return lhs.stdErrHandle == rhs.stdErrHandle && lhs.stdOutHandle == rhs.stdOutHandle;
-        }
+        friend constexpr bool operator==(const ConsoleState& lhs, const ConsoleState& rhs);
```

Comment:
I am not sure to understand this change?

Reply:
This was me shuffling around the code while investigating why sometimes the console restauration was not behaving well for Linux apps. The only difference is that this operator now allows comparison in compile time, but it is very unlikely that will ever make sense. So, in practice nothing changes. I should have dropped it, but I forgot.

Reply:
Removed now.